api/v1/applicationlayer_types.go

```diff
@@ -22,15 +22,20 @@ import (
 
 // ApplicationLayerSpec defines the desired state of ApplicationLayer
 type ApplicationLayerSpec struct {
+	// Specification for WAF (Web Application Firewall).
```

Comment:
I know the other comment is bare-bones as well, but remember this is user-facing documentation so we really need more description for this.

"WebApplicationFirewall controls whether or not ModSecurity enforcement is enabled for the cluster. When enabled, Services may opt-in to having ingress traffic examed by ModSecurity"

Or something like that.

Reply:
done.